src/components/Playlist.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import styles from '../styles/Playlist.module.css';
+import Track from '../components/Track.jsx';
+import { obtainUserId, addToPlaylist } from '../Constants/httpRequests.js';
+
+export default function Playlist({ trackUris, tracks, onRemoveResult }) {
```

Comment:
In our next meeting lets talk about a way to combine trackUris and tracks into a single parameter so you don't have to manage two separate lists